src/less/textbox/ds6/textbox-base.less

```diff
@@ -0,0 +1,164 @@
+// Textbox text should NOT scale with root font-size. All font-size defined with px.
+
+.textbox {
+    position: relative;
+}
+
+span.textbox {
+    display: inline-block;
+}
+
+input.textbox__control,
+textarea.textbox__control {
+    .placeholder;
+
+    appearance: none;
+    background: white;
+    border: 1px solid @ds6-color-g205-gray;
+    box-sizing: border-box;
+    color: @ds6-color-g206-gray;
+    font-family: inherit;
+    font-size: inherit;
+    height: 40px;
+    line-height: 1.15; // normalized line-height
+    margin: 0; // Remove the margin in Firefox and Safari.
+    padding: 0 16px 0 16px;
+
+    &[disabled] {
+        .placeholder-disabled;
+
+        border-color: @ds6-color-g203-gray;
+    }
+
+    &[readonly] {
+        border-color: @ds6-color-g203-gray;
+    }
+
+    &[aria-invalid="true"] {
+        border-color: @ds6-color-p013-red;
+    }
+
+    &:focus {
+        border-color: @ds6-color-p002-blue;
+        outline: 0;
+    }
+}
+
+textarea.textbox__control {
+    min-height: 200px;
+    overflow: auto;
+    padding-bottom: 16px;
+    padding-top: 16px;
+    resize: vertical;
+    vertical-align: middle;
+}
+
+span.textbox__icon,
+.textbox__icon {
+    display: inline-flex;
+    height: 100%;
+    pointer-events: none;
+    position: absolute;
+    top: 0;
+
+    &::before {
+        align-self: center;
+    }
+}
+
+span.textbox__icon {
+    color: @ds6-color-g204-gray;
+    font-size: 14px; // icon shouldn't scale with font-size cascade
+}
+
+.textbox__icon {
+    fill: @ds6-color-g204-gray;
+    width: 1em;
```

Comment:
Elsewhere you state that the font shouldn't scale, yet this uses `em`. Should this be `px` to match the `<span>` version?